### PR TITLE
PasswordChange: Use builtin password validator

### PR DESCRIPTION
**Pull request by y3n4**
Use Symfonys builtin validators for PasswordChange

### CODE REVIEW

File: features/user.feature [hunk 1]
```diff
@@ -23,8 +23,8 @@ Feature: User
     And I am on "/account"
     And I fill in the following:
       | password_change_password             | asdasd       |
-      | password_change_plainPassword_first  | P4ssW0rd!!!1 |
-      | password_change_plainPassword_second | P4ssW0rd!!!1 |
+      | password_change_newPassword_first  | P4ssW0rd!!!1 |
+      | password_change_newPassword_second | P4ssW0rd!!!1 |
```
Comment (0x46616c6b) [inline features/user.feature:27]:
```suggestion
      | password_change_newPassword_first    | P4ssW0rd!!!1 |
      | password_change_newPassword_second   | P4ssW0rd!!!1 |
```

File: src/Form/Model/PasswordChange.php [hunk 2]
```diff
@@ -2,13 +2,37 @@
...
+    #[Assert\NotCompromisedPassword(skipOnError: 'true')]
+    #[Assert\NotIdenticalTo(propertyPath: 'password', message: 'form.identical-passwords')]
+    private string $newPassword;
+
+    public function getPassword(): ?string
+    {
+        return $this->password;
+    }
+
+    public function setPassword(?string $password): void
```
Comment (0x46616c6b) [inline src/Form/Model/PasswordChange.php:24]:
```suggestion
    public function setPassword(string $password): void
```

I think we should be unable to `null` the password

File: src/Form/Model/PasswordChange.php [hunk 3]
```diff
@@ -2,13 +2,37 @@
...
+    {
+        $this->password = $password;
+    }
+
+    public function getNewPassword(): ?string
+    {
+        return $this->newPassword;
+    }
+
+    public function setNewPassword(?string $newPassword): void
```
Comment (0x46616c6b) [inline src/Form/Model/PasswordChange.php:34]:
```suggestion
    public function setNewPassword(string $newPassword): void
```

I think we should be unable to `null` the newPassword

**Review by 0x46616c6b — CHANGES_REQUESTED** [review]

File: features/user.feature (hunk 1 above)
Comment (y3n4) [reply features/user.feature]: ✅

File: src/Form/Model/PasswordChange.php (hunk 2 above)
Comment (y3n4) [reply src/Form/Model/PasswordChange.php]: ✅

**Review by 0x46616c6b — APPROVED** [review]
Really nice improvement!

**Pull request merged**